backend/src-common/src/main/java/org/eclipse/sw360/datahandler/db/ProjectDatabaseHandler.java

```diff
@@ -447,80 +468,138 @@ public RequestStatus updateProjectVulnerabilityRating(ProjectVulnerabilityRating
+                }
+            });
+        }
+
+        // add own releases to result if they are not just "REFERRED"
+        if (project.isSetReleaseIdToUsage()) {
+            project.getReleaseIdToUsage().entrySet().stream().forEach(e -> {
+                if (!ReleaseRelationship.REFERRED.equals(e.getValue().getReleaseRelation())) {
+                    releaseIds.add(e.getKey());
+                }
+            });
+        }
+
+        return releaseIds;
+    }
+
     private void sendMailNotificationsForNewProject(Project project, String user) {
-        mailUtil.sendMail(project.getProjectResponsible(),
-                MailConstants.SUBJECT_FOR_NEW_PROJECT,
-                MailConstants.TEXT_FOR_NEW_PROJECT,
-                SW360Constants.NOTIFICATION_CLASS_PROJECT, Project._Fields.PROJECT_RESPONSIBLE.toString(),
-                project.getName(), project.getVersion());
-        mailUtil.sendMail(project.getProjectOwner(),
-                MailConstants.SUBJECT_FOR_NEW_PROJECT,
-                MailConstants.TEXT_FOR_NEW_PROJECT,
-                SW360Constants.NOTIFICATION_CLASS_PROJECT, Project._Fields.PROJECT_OWNER.toString(),
-                project.getName(), project.getVersion());
-        mailUtil.sendMail(project.getLeadArchitect(),
-                MailConstants.SUBJECT_FOR_NEW_PROJECT,
-                MailConstants.TEXT_FOR_NEW_PROJECT,
-                SW360Constants.NOTIFICATION_CLASS_PROJECT, Project._Fields.LEAD_ARCHITECT.toString(),
-                project.getName(), project.getVersion());
-        mailUtil.sendMail(project.getModerators(), user,
-                MailConstants.SUBJECT_FOR_NEW_PROJECT,
-                MailConstants.TEXT_FOR_NEW_PROJECT,
-                SW360Constants.NOTIFICATION_CLASS_PROJECT, Project._Fields.MODERATORS.toString(),
-                project.getName(), project.getVersion());
-        mailUtil.sendMail(project.getContributors(), user,
-                MailConstants.SUBJECT_FOR_NEW_PROJECT,
-                MailConstants.TEXT_FOR_NEW_PROJECT,
-                SW360Constants.NOTIFICATION_CLASS_PROJECT, Project._Fields.CONTRIBUTORS.toString(),
-                project.getName(), project.getVersion());
-        mailUtil.sendMail(project.getSecurityResponsibles(), user,
-                MailConstants.SUBJECT_FOR_NEW_PROJECT,
-                MailConstants.TEXT_FOR_NEW_PROJECT,
-                SW360Constants.NOTIFICATION_CLASS_PROJECT, Project._Fields.SECURITY_RESPONSIBLES.toString(),
-                project.getName(), project.getVersion());
-        mailUtil.sendMail(SW360Utils.unionValues(project.getRoles()), user,
-                MailConstants.SUBJECT_FOR_NEW_PROJECT,
-                MailConstants.TEXT_FOR_NEW_PROJECT,
-                SW360Constants.NOTIFICATION_CLASS_PROJECT, Project._Fields.ROLES.toString(),
-                project.getName(), project.getVersion());
+        mailUtil.sendMail(project.getProjectResponsible(), MailConstants.SUBJECT_FOR_NEW_PROJECT,
```

Comment:
did you not like my carefully manually formatted code so much, or is it just the auto-formatting you applied indiscriminately?

Reply:
Ooops, I just moved the methods to the end of the class after rebasing and that way autoformat thought I "edited" those lines :-/ Will fix it in a second.